packages/vscode-extension/schemas/proc_grps.schema.json

```
@@ -53,16 +58,23 @@
               "check-margins": {
                 "type": "boolean",
                 "description": "Whether to check for margins in the PL/I code."
+              },
+              "instruction-counter-limit": {
+                "type": "number",
+                "description": "Maximum number of tries to run an instructions during preprocessing."
```

Minor: Not entirely accurate. I would also add hint about the default.
```suggestion
                "description": "Maximum number of times to run an instructions during preprocessing. 5000 by default."
```